### PR TITLE
Blog

### PR DESCRIPTION
**Pull request by anushadatta**
Added a blog section to the main page, and created a separate HTML page for the blog. Also refactored code to reduce duplication of code, by sharing common CSS styles.

### CODE REVIEW

File: portfolio/src/main/webapp/blog.html [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+            <a href="https://github.com/anushadatta" target="_blank"><i class="fa fa-github"></i></a>
+
+            <a href="https://medium.com/@anushadatta" target="_blank"><i class="fa fa-medium"></i></a>
+
+        </div>
+    </section>
+
+    <!-- Footer -->
+    <footer class="center bg-dark">
```
Comment (rashikaranpuria) [inline portfolio/src/main/webapp/blog.html:58]: Optional, If you have several web pages with common header footer or other sections, consider declaring them at one place and re-use.
Remember the principle DRY-Don't repeat yourself.
Although I don't blieve we can do this using pure html, we may need to use server side language like PHP or some framework for this kind of templating. Hence this comment is optional.

File: portfolio/src/main/webapp/blog.html [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+        <p>Anusha Datta &copy; 2020</p>
+    </footer>
+
+</body>
+
+</html>
```
Comment (rashikaranpuria) [inline portfolio/src/main/webapp/blog.html:64]: It's suggested to have newline at the end of file. Notice the red error icon on your last line.

File: portfolio/src/main/webapp/styles/blog.css [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+    height: 50px;
+    width: 50px;
+}
+
+.content .title {
+    font-size: 5rem;
+    margin-top: 1rem;
+    font-weight: bold;
+    margin: 0px;
+}
```
Comment (rashikaranpuria) [inline portfolio/src/main/webapp/styles/blog.css:45]: ditto

File: portfolio/src/main/webapp/styles/common.css [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+
+/* Website Footer */
+footer {
+    padding: 2.2rem;
+  
+}
+
+footer p {
+    margin: 0;
+}
```
Comment (rashikaranpuria) [inline portfolio/src/main/webapp/styles/common.css:72]: ditto

File: portfolio/src/main/webapp/styles/main-page.css [hunk 5]
```diff
@@ -103,6 +63,7 @@ body {
     transition: all 1000ms;
 }
```
Comment (rashikaranpuria) [inline portfolio/src/main/webapp/styles/main-page.css:65]: remove one blank line

File: portfolio/src/main/webapp/styles/main-page.css [hunk 6]
```diff
@@ -64,6 +23,7 @@ body {
...
 
+
```
Comment (rashikaranpuria) [inline portfolio/src/main/webapp/styles/main-page.css:26]: remove one blank line

File: portfolio/src/main/webapp/styles/main-page.css [hunk 7]
```diff
@@ -164,6 +118,7 @@ hr {
     margin: auto;
 }
 
+
```
Comment (rashikaranpuria) [inline portfolio/src/main/webapp/styles/main-page.css:121]: ditto, everywhere else below. No need to add two new line

File: portfolio/src/main/webapp/styles/main-page.css [hunk 8]
```diff
@@ -263,26 +219,45 @@ hr {
...
 }
 
-/* Contact icons */
-#contact i {
-    font-size: 30px;
-    margin: 15px;
-    color: #333;
+
+/* BLOG POSTS */
+
```
Comment (rashikaranpuria) [inline portfolio/src/main/webapp/styles/main-page.css:224]: ditto

File: portfolio/src/main/webapp/styles/main-page.css [hunk 9]
```diff
@@ -327,3 +301,14 @@ footer p {
...
+.blogimg {
+    max-width: 400px;
+}
+
+.blogimg img {
+    max-width: 50%;
+    max-height: 50%;
+    display: inline-block;
+    height: auto;
+}
```
Comment (rashikaranpuria) [inline portfolio/src/main/webapp/styles/main-page.css:314]: ditto

File: portfolio/src/main/webapp/blog.html (hunk 1 above)
Comment (anushadatta) [reply portfolio/src/main/webapp/blog.html:58]: Yes, I'm not sure I can enforce that here, but I'll definitely keep that in mind for any future projects I do. Thank you for that notable insight!

File: portfolio/src/main/webapp/blog.html (hunk 2 above)
Comment (anushadatta) [reply portfolio/src/main/webapp/blog.html:64]: I was absolutely unaware of this, so glad it came up!!! Just researched it in more detail, and can now appreciate the rationale behind this practice. 👍 

I shall fix this in all files necessary.

File: portfolio/src/main/webapp/styles/blog.css (hunk 3 above)
Comment (anushadatta) [reply portfolio/src/main/webapp/styles/blog.css:45]: A "sure, i gotcha" for all the "ditto"s below. 🎉

File: portfolio/src/main/webapp/styles/main-page.css (hunk 6 above)
Comment (anushadatta) [reply portfolio/src/main/webapp/styles/main-page.css:26]: Now only to pick up this extra line & give it to the other files that need it haha.

**Pull request merged**